form-link/src/main/java/com/ritense/formlink/domain/impl/formassociation/CamundaFormAssociation.java

```diff
@@ -20,6 +20,7 @@
 import com.fasterxml.jackson.databind.node.ObjectNode;
 import com.ritense.formlink.domain.FormAssociation;
 import com.ritense.formlink.domain.FormLink;
+import kotlin.jvm.Transient;
```

Comment:
Is this the class you wanted to apply? Not `javax.persistence.Transient` for example? Not sure what this is supposed to fix.

Reply:
Sonar has a problem with this fields in a Serialized class  that has no @Transcient or @Serializable annotation. Sonar says that : it could cause program crashes  or is a security risk while opening the door to attackers. No problem here to roll this change back.